tap_stripe/__init__.py

```diff
@@ -698,6 +698,17 @@ def sync_event_updates(stream_name):
                 event_resource_metadata = metadata.to_map(
                     Context.get_catalog_entry(stream_name)['metadata']
                 )
+
+                # Filter out line items with null ids
+                if events_obj.get('data', {}).get('object', {}).get('lines', {}).get('data'):
+
+                    invoice_obj = events_obj.get('data').get('object')
+                    line_items = invoice_obj.get('lines').get('data')
```

Comment:
Would probably be cleaner to get these outside the loop and just check if `lines` is `None` to get in (along with the above comment).

Reply:
Which loop do you mean?

Reply:
Sorry, was typing fast 😅. I mean the conditional, not loop.